src/Practitioner/ReviewPatients/HomeTopBar.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Box, Grid, IconButton } from '@material-ui/core';
+import { PageLabelTitle } from '../../Components/Shared/PageLabel';
+import { Search, Add } from '@material-ui/icons';
+import Colors from '../../Basics/Colors';
+import { Link } from 'react-router-dom';
+
+export default function TopBar() {
+  const { t } = useTranslation('translation');
+  return (
+    <Box bgcolor="white" borderBottom="solid 1px lightgray" padding=".5em 1em">
+      <Grid alignItems="center" container>
+        {/* Why does it say all patients? */}
```

Comment:
This is prolly cause its an existing translation we are using before it was used in this context, good call tho

Reply:
Changed to 'Daily Reports'